psx/src/panic.rs

```diff
@@ -53,5 +55,6 @@ fn panic(info: &core::panic::PanicInfo) -> ! {
         }
     };
     // Both min_panic and no_panic end execution here
+    #[cfg(any(feature = "min_panic", feature = "no_panic"))]
```

Comment:
Please remove this `cfg` attribute since it'll make this function fail to compile when neither feature is enabled. The other changes to this file are fine.

Reply:
I agree that it doesn't look very good, but removing it will give us a warning that the loop statement is unreachable by default. since if no features are enabled, the rendering loop will be present